teamviewer/artifactfile.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Teamviewer",
+  "description": "Installs Teamviewer - desktop sharing / collaboration tool",
+  "tags": [
+    "Teamviewer",
+    "Collaboration"
+  ],
+    "iconUri": "https://chocolatey.org/content/packageimages/teamviewer.11.0.66695.png",
```

Comment:
I've found that once chocolately packages get new versions, the URIs to an icon like this is broken. Instead refer to one that won't likey change, or since you've add a png logo in this dir, refer to the raw uri to it here.